boards/arm/arty/doc/index.rst

```diff
@@ -133,6 +133,12 @@ or:
         pld load 0 m3_for_arty_a7_reference.bit;\
         shutdown"
 
+.. note::
+
+   The pre-built FPGA bitstream only works for Arty boards equipped with an
+   Artix-35T FPGA. For other Arty variants (e.g. the Arty A7-100) the bitstream
```

Comment:
s/A7-100/A7-100T/

Reply:
Not according to https://store.digilentinc.com/arty ?

Reply:
Hmn, they're not entirely consistent.  From https://store.digilentinc.com/arty-a7-artix-7-fpga-development-board-for-makers-and-hobbyists/:

> Arty A7 comes in two FPGA variants: Arty A7-35T features Xilinx XC7A35TICSG324-1L. Arty A7-100T features the larger Xilinx XC7A100TCSG324-1. Arty A7 variants have the features listed below:

I got thrown by having `Artix-35T` in the same sentence.  If you're consistent in using `A7-35` everywhere, then `A7-100` is...consistent.

Removed nak, and thanks for doing this.